webpack/js/modules.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (ENV, { SRC }) => {
+  const eslintLoader = (ENV === 'production')
+    ? {
+      test: /\.jsx?$/,
+      enforce: 'pre',
+      loader: 'eslint-loader',
+      options: {
+        filePath: SRC,
+      },
+    }
+    : null;
+
+  return {
+    rules: [
+      eslintLoader,
+      {
+        // .js and .jsx files are caught
+        test: /\.jsx?$/,
+        exclude: /(node_modules|thirdparty)/,
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            ['es2015', { modules: false }],
```

Comment:
interesting, I hadn't run into that during my research for upgrading...
I had only found the `{ modules: false }` config, which stops babel from transforming all imports/exports to commonjs
I'll look into this a bit more, thanks!